specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4143,6 +4181,24 @@
       "type": "string",
       "in": "path",
       "x-ms-skip-url-encoding": true
+    },
+    "RecognitionModel": {
+      "name": "RecognitionModel",
+      "description": "Recognition model name.\nRecognition model is used when the face features are extracted, so a recognition model version could be provided when performing a Detection.\n",
```

Comment:
I will refine these descriptions finally anyway. But please add for the case of user's not specifying recognition model here.

Reply:
Added.